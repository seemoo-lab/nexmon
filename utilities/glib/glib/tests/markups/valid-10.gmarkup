<foo 
 bar="baz" 
bar2 = "baz2" 
bar3 =
"baz3"
>data</foo>
