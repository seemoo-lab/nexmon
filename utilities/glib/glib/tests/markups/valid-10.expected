ELEMENT 'foo'
bar="baz"
bar2="baz2"
bar3="baz3"
  TEXT 'data'
END 'foo'
